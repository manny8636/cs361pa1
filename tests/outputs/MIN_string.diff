--- outputs/MIN_string.txt	2022-02-13 13:37:45.547538218 -0500
+++ expected/MIN_string.txt	2021-01-25 15:27:24.000000000 -0500
@@ -1 +1,2 @@
+VALUE [string]: 'I am a simple string'
 Success!
