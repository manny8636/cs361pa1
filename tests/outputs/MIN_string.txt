Parsing ../data/string.txt failed
