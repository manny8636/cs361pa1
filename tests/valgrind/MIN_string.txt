==146677== Memcheck, a memory error detector
==146677== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==146677== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==146677== Command: ../json -v ../data/string.txt
==146677== 
Parsing ../data/string.txt failed
==146677== 
==146677== HEAP SUMMARY:
==146677==     in use at exit: 23 bytes in 1 blocks
==146677==   total heap usage: 4 allocs, 3 frees, 16,879 bytes allocated
==146677== 
==146677== LEAK SUMMARY:
==146677==    definitely lost: 23 bytes in 1 blocks
==146677==    indirectly lost: 0 bytes in 0 blocks
==146677==      possibly lost: 0 bytes in 0 blocks
==146677==    still reachable: 0 bytes in 0 blocks
==146677==         suppressed: 0 bytes in 0 blocks
==146677== Rerun with --leak-check=full to see details of leaked memory
==146677== 
==146677== For lists of detected and suppressed errors, rerun with: -s
==146677== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
