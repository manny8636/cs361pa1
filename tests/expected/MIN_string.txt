VALUE [string]: 'I am a simple string'
Success!
